code/modules/surgery/organs/organ_external.dm

```diff
@@ -487,14 +487,43 @@ Note that amputating the affected organ does in fact remove the infection from t
 		//Open wounds can become infected
 		if(owner.germ_level > W.germ_level && W.infection_check())
 			W.germ_level++
-
+	for(var/obj/item/organ/external/O in owner)
+		//incisions make it easier to get an infection
+		if(O.open)
+			spread_germs_by_incision(O)
 	if(antibiotics < 5)
 		for(var/datum/wound/W in wounds)
 			//Infected wounds raise the organ's germ level
 			if(W.germ_level > germ_level)
 				germ_level++
 				break	//limit increase to a maximum of one per second
 
+/proc/spread_germs_by_incision(obj/item/organ/external/E)
+	if(!istype(E))
+		return
+
+	if(!(E.status & ORGAN_ROBOT))
+		var/germ_count = 0
+		var/area/room = get_area(E)
+		for(var/mob/living/carbon/human/H in room)
+			if(!(NO_BREATH in H.mutations) || !(H.species.flags & NO_BREATH))
+				if(H.wear_mask)
+					//wearing a mask helps preventing people from breathing cooties into open incisions
+					germ_count+=H.germ_level/200
+				else
+					germ_count+=H.germ_level/100
+
+		for(var/obj/effect/decal/cleanable/M in room)
```

Comment:
Given how prevalent dirt is and how hard it is to see, at first, this is going to cause a lot of problem and lead to spraying down of the entire surgery room every single time.